control-plane-operator/controllers/hostedcontrolplane/kas/config.go

```diff
@@ -191,7 +191,8 @@ func generateConfig(p KubeAPIServerConfigParams, version semver.Version) *kcpv1.
 	args.Set("service-account-lookup", "true")
 	args.Set("service-account-signing-key-file", cpath(kasVolumeServiceAccountKey().Name, pki.ServiceSignerPrivateKey))
 	args.Set("service-node-port-range", p.NodePortRange)
-	args.Set("shutdown-delay-duration", "10s")
+	args.Set("shutdown-delay-duration", "70s")
```

Comment:
I don't see how this is related to ImagePullPolicy. Please read our contributing guide: https://hypershift-docs.netlify.app/contribute/

/hold